### ISSUE TITLE
Infill Transitioning

### ISSUE THREAD

**Issue by cfelicio** [spegelius/filaswitch#33]
Not sure if even remotely possible to implement something like this, but quite amazing feature:

https://www.reddit.com/r/3Dprinting/comments/7t5zx8/mosaic_introduces_infill_transitioning/

**Comment by spegelius** [comment]
Hmm, interesting. Yes it's possible but requires some work. Will require more thorough analysis of the gcode and some calculation about infill length and if additional purge is needed...

**Comment by wbarber69** [comment]
this can only work if your part has enough infill to do the transition

**Comment by spegelius** [comment]
That's true. The implementation would still need purge tower as backup location for cases when there's not enough infill for the purge.
Other possibility that I've thought about is using another model as purge target. The model would have some requirements for optimal usage and of cource be of such use that having wild color transitions do not matter. Combination of infill + surrogate model would be possible, but does require considerable amount of work and there might a issues I've not thought about...
